server/src/main/java/com/project/server/Application.java

```diff
@@ -30,6 +36,36 @@ public CommandLineRunner demo(UserRepository users, PermissionRepository permiss
                                   ProjectRepository projectRepo, ToolRepository toolRepo, ScanRepository scanRepo) {
         return (args) -> {
 //            Users
+
+        try{    
+	
+        // Run a shell script
+
+        Process proc = Runtime.getRuntime().exec("docker run --rm  aquasec/trivy -f json --light python:3.4-alpine");
+        System.out.println("Success!");
+
+        BufferedReader stdInput = new BufferedReader(new 
+     InputStreamReader(proc.getInputStream()));
+
+BufferedReader stdError = new BufferedReader(new 
+     InputStreamReader(proc.getErrorStream()));
+
+// Read the output from the command
+System.out.println("Here is the standard output of the command:\n");
+String s = null;
+while ((s = stdInput.readLine()) != null) {
+    System.out.println(s);
+}
+
+// Read any errors from the attempted command
+System.out.println("Here is the standard error of the command (if any):\n");
+while ((s = stdError.readLine()) != null) {
+    System.out.println(s);
+}
+
+	} catch (IOException e) {
+		e.printStackTrace();
+	}
```

Comment:
Jedna bezpańska linia kodu nieopakowana konkretną metodą = jedna umierająca szara komórka podczas czytania tego. Nie róbcie tego mi i sobie. Wiem, że to testowe, ale nie umniejsza to syfu, jaki się tu robi.